### PR TITLE
Add CI checks for Terraform

### PR DESCRIPTION
**Pull request by JesterOrNot**
Let's make sure the scripts don't break :)

### CODE REVIEW

**Comment by csweichel** [comment]
The checks make total sense to me.

Could you please make them run in parallel to the deployment? This way we don't loose time on the checks which are inconsequential for the helm dev-staging deployment.

File: .werft/build.js [hunk 1]
```diff
@@ -62,6 +62,26 @@ async function build(context, version) {
...
+        exec(`cd install/${i} && terraform init -backend=false`, { slice: `install/${i}`})
+        werft.log(`install/${i}`, "\x1b[1;34mCheck Style\x1b[m")
+        exec(`cd install/${i} && terraform fmt -recursive -check`, {slice: `install/${i}`})
+        werft.log(`install/${i}`, "\x1b[1;34mValidating\x1b[m")
+        exec(`cd install/${i} && terraform validate`, { slice: `install/${i}`})
+        werft.done(`install/${i}`)
+      } catch (err) {
+        werft.fail(`install/${i}`, err)
+      }
+    }
```
Comment (meysholdt) [inline .werft/build.js:84]: can you move the new code into a method? something like `validateTerraform()`.

Comment (JesterOrNot) [reply .werft/build.js]: Done!

**Review by meysholdt — APPROVED** [review]
Can you rebase to resolve the conflict? Besides this, it looks good from my side. Would be nice if @csweichel could approve, too, because this version looks like some of his advice went into it.

**Pull request merged**